toolkit/tools/pkg/imagecustomizerlib/artifactsinputoutput_test.go

```diff
@@ -0,0 +1,125 @@
+package imagecustomizerlib
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/file"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestOutputAndInjectArtifacts(t *testing.T) {
+	baseImage := checkSkipForCustomizeImage(t, baseImageTypeCoreEfi, baseImageVersionAzl3)
+
+	testTempDir := filepath.Join(tmpDir, "TestOutputAndInjectArtifacts")
+	buildDir := filepath.Join(testTempDir, "build")
+	outImageFilePath := filepath.Join(testTempDir, "image.raw")
+	configFile := filepath.Join(testDir, "artifacts-output.yaml")
```

Comment:
You should use `testTempDir` for both `configFile` and `outputArtifactsDir`. That way, the files will be automatically cleaned up at the end of the test.